source/GameInterface/Services/Time/Handlers/TimeControlHandler.cs

```diff
@@ -29,6 +29,10 @@ private void Handle(MessagePayload<SetTimeControlMode> obj)
     {
         var payload = obj.What;
         CampaignTimeControlMode newTimeMode = (CampaignTimeControlMode)payload.NewTimeMode;
+
+        if (newTimeMode == CampaignTimeControlMode.StoppablePlay) { newTimeMode = CampaignTimeControlMode.UnstoppablePlay; }
+        if (newTimeMode == CampaignTimeControlMode.StoppableFastForward) { newTimeMode = CampaignTimeControlMode.UnstoppableFastForward; }
+
```

Comment:
I don't understand how these changes solve the listed issue. The seems to only convert our interface time controls with the games

Reply:
Every time Change in game does a isNot check for unstoppable time. If we always set it as unstoppable we need less patches.

Reply:
Except setting it in the controls, from what i found